src/clj/game/cards/assets.clj

```diff
@@ -1385,21 +1385,27 @@
                                       (effect-completed state side eid)))))
            :cancel-effect (effect (system-msg "declines to use Moon Pool to reveal any cards in Archives")
                                   (effect-completed eid))}]
-      {:abilities [{:prompt "Trash up to 2 cards from HQ"
-                    :label "Trash up to 2 cards from HQ"
-                    :cost [:remove-from-game]
-                    :async true
-                    :choices {:card #(and (corp? %)
-                                          (in-hand? %))
-                              :max 2}
-                    :msg (msg "trash " (count targets) " cards from HQ ")
-                    :effect (req (wait-for (trash-cards state :corp targets {:cause-card card})
-                                           (continue-ability
-                                             state side
-                                             moon-pool-reveal-ability
-                                             card nil)))
-                    :cancel-effect (effect (system-msg "declines to use Moon Pool to trash any cards from HQ")
-                                           (continue-ability moon-pool-reveal-ability card nil))}]})))
+      (let [moon-pool-discard-ability
```

Comment:
There's no need for a second let block, move this into the enclosing let block.

Reply:
Thanks, I agree. It was a remnant of when I was messing with letfn